components/infobox/extensions/wikis/rocketleague/years_active.lua

```diff
@@ -7,20 +7,96 @@
 --
 
 local Class = require('Module:Class')
+local Lpdb = require('Module:Lpdb')
 local Lua = require('Module:Lua')
+local Set = require('Module:Set')
+local Table = require('Module:Table')
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
 
 local CustomActiveYears = Lua.import('Module:YearsActive/Base', {requireDevIfEnabled = true})
 
 -- wiki specific settings
 CustomActiveYears.defaultNumberOfStoredPlayersPerPlacement = 6
 CustomActiveYears.additionalConditions = ''
 
+local _TALENT_POSITIONS = {
```

Comment:
Can we skip the filtering based on text in the bc table?

Reply:
There's no disambiguation between roles other than the text that they are

Reply:
What I mean is grabbing all broadcast table insertions as part of years active as broadcaster; instead of just a subset. I foresee this causing issues in the future.

Reply:
Not all roles in the broadcast table are talent, e.g. `Guest`

Reply:
Then they shouldn't in the in the Broadcast _**TALENT**_ table?

Reply:
The broadcaster cards across many wikis seem to commonly have anyone featured/credited in the broadcast, such as guests, observers, producers, etc.

Reply:
Wikis that manually input the years-active also don't seem to class these roles (guests, observers, producers) as talent